xyz.xclicker.xclicker.yml

```diff
@@ -0,0 +1,48 @@
+app-id: xyz.xclicker.xclicker
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+command: xclicker
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --socket=ssh-auth
+  - --filesystem=~/.config
```

Comment:
Apologies if we discussed it before, but can you remind me what are these two needed for? `ssh-auth` for automatic clickers sounds fishy, while `~/.config` is both wrong and shouldn't be needed, your application should respect $XDG_CONFIG_HOME instead.

Reply:
If I remember it correctly, Ssh-auth is there because else gtk threw warnings. For the config, wdym? Should I change it in the application to not use .config and instead xdg_config_home?

Reply:
If these were warnings, I don't think you need ssh-auth for anything, especially if your app doesn't use ssh. About the config, yes, $XDG_CONFIG_HOME is preferred, but there's a large chance that it is handled automatically by GTK.

Reply:
So, what should I set the filesystem to?  `--filesystem=xdg-config-home` is not working. And if xdg-config-home is unset, how would I allow access to ~/.config instead?